src/uwtools/cli/atparse_to_jinja2.py

```diff
@@ -53,8 +55,23 @@ def parse_args(args: List[str]) -> Namespace:
         help="Path to new Jinja2 template",
         metavar="FILE",
     )
+    optional.add_argument(
+        "-q",
+        "--quiet",
+        action="store_true",
+        help="Print no logging messages",
+    )
+    optional.add_argument(
+        "-v",
+        "--verbose",
+        action="store_true",
+        help="Print all logging messages.",
+    )
```

Comment:
In UW-343, a **lot** of duplication will be removed by coalescing the CLI logic into a single module. For now, to keep things consistent on `develop`, add the `--quiet` and `--verbose` flags to the `arparse-to-jinja2` program so that logging can be correctly configured.